src/or/circuitmux_ewma.c

```diff
@@ -597,24 +599,48 @@ cell_ewma_to_circuit(cell_ewma_t *ewma)
    rescale.
  */
 
-/** Given a timeval <b>now</b>, compute the cell_ewma tick in which it occurs
- * and the fraction of the tick that has elapsed between the start of the tick
- * and <b>now</b>.  Return the former and store the latter in
- * *<b>remainder_out</b>.
+/**
+ * Initialize the system that tells which ewma tick we are in.
+ */
+STATIC void
+cell_ewma_initialize_ticks(void)
+{
+  if (ewma_ticks_initialized)
+    return;
+  monotime_coarse_get(&start_of_current_tick);
+  crypto_rand((char*)&current_tick_num, sizeof(current_tick_num));
```

Comment:
Oh wow, I guess this is an upgraded security feature that we didn't have? That is starting the EWMA tick number at a random value! :) +1

Reply:
I don't *think* it's a security feature, since we don't actually expose tick numbers.

My motivation here was to make sure that we would observe tick rollover sometimes, even on relays that aren't up for long enough to overflow the tick counter.  Otherwise, overflow would be very difficult to test.